cg/apps/microsalt/fastq.py

```diff
@@ -36,12 +36,12 @@ def __init__(self, config):
         super().__init__(config)
         self.root_dir = config["usalt"]["root"]
 
-    def link(self, case: str, sample: str, files: List):
+    def link(self, ticket: int, sample: str, files: List):
```

Comment:
List of what? Could you also add return type? Suggestion
```suggestion
    def link(self, ticket: int, sample: str, files: List[Path]) -> None:
```
I think we should aim for a situation where we can run mypy checks on all our code

Reply:
fixed